### DISCUSSION TITLE
Release 2.0.1

### DISCUSSION THREAD

**Discussion by cviebrock** [cviebrock/sequel-pro-laravel-export#41] [General]
# Version 2.0.1

This version adds support for foreign keys.  All tables are created first, then separate migrations to add the foreign keys to those tables are created next.  This ensures that all the tables are in place first before trying to reference them.  Thanks to @rootrus for his PR work on this!

I've also added support for converting `->bigIncrements("column")` to `->id("column")` in order to make the migrations more "Laravel-y".  If the column is called `id`, it will simplify it even more by just using `->id()` with no column name in the migration.  Again, thanks to @rootrus for his work on this.

Finally, I've made a few minor changes to the handling of empty comments.

Remember, the `.spbundle` file in the release is for Sequel Pro users, and the `.sabundle` file is for Sequel Ace users (version 3.0.0 and higher only).

**Comment by pemedina** [comment]
Hello:

I see there's a tag 2.0.1 but i can see in master branch:

```
 /**
     * @var string
     */
    protected $version = '1.8.1';
```
 ¿Should this string be updated to reflect latest version?. Our migrations are created with:

```
/**
 * Migration auto-generated by Sequel Pro/Ace Laravel Export (1.8.1)
 * @see https://github.com/cviebrock/sequel-pro-laravel-export
 */
```

and this is kinda confusing.

**Comment by cviebrock** [comment]
You are right ... this was an oversight on my part.

I will fix it and release a 2.0.2.

**Reply by pemedina** [reply]
Thanks! We'll wait for it.

Regards,

**Reply by pemedina** [reply]
Ops..i see it's already there. Thanks!
